spec/db/schema_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'db:check_for_sensitive_columns' do
+  around do |ex|
+    ex.run
+  rescue SystemExit
+  end
+
+  before :all do
+    Rake.application.rake_require 'tasks/column_comment_checker'
+    Rake::Task.define_task(:environment)
+  end
+
+  let(:task) { Rake::Task['db:check_for_sensitive_columns'] }
+
+  it 'checks for columns with sensitivity comments' do
+    expect { task.execute }.to output(/All columns have sensitivity comments./).to_stdout
+  end
```

Comment:
should we update the description or failure message to have example so that when this fails, people have clear instructions to follow to add comments in their migrations and fix the errors?

something like
```
in your migration file, add `comment: "sensitive=false"` to all columns added
````

Reply:
Yes, I suppose I either shouldn't suppress puts or leave a comment in the tests to run the task locally.

Reply:
maybe there's a way to capture the STDOUT and then put that as the test expectation? This would just print the string it got instead

```suggestion
  it 'checks for columns with sensitivity comments' do
    stdout = StringIO.new('')
    stub_const(STDOUT, stdout)

    task.execute
    
     expect(stdout.string).to match(/All columns have sensitivity comments./)
  end
```

Reply:
TIL if you expect output after a rake task executes with an exit(1)... it will always pass.

Reply:
TBH I'm not 100% sure that a rake task is the best approach for this, especially if we want to test exit codes and things, that gets very challenging. Just a spec that checks the columsn would be fine

Reply:
Yeah, I don't think we want to test the exit codes. The rake task is nice to run as a one-off. I think it's in a good spot now.